lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -312,19 +270,10 @@ sub _cache_sync {
 sub asset_lookup {
     my ($self, $asset) = @_;
 
-    my $results;
-    eval {
-        my $db = $self->sqlite->db;
-        my $tx = $db->begin('exclusive');
-        $results = $db->select('assets', [qw(filename etag last_use size)], {filename => $asset});
-        $tx->commit;
-    };
-    if ($@) {
-        return undef;
-    }
+    my $results = $self->sqlite->db->select('assets', [qw(filename etag last_use size)], {filename => $asset});
 
     if ($results->arrays->size == 0) {
-        $self->log->info("CACHE: Purging non registered $asset");
+        $self->log->info("Purging $asset because the asset is not registered");
```

Comment:
I am in favor of putting values used within strings within quotes in the output to easily see where the value starts or even recognize empty strings, regardless if they are programattically prevented, e.g. example output: `"Purging '' because the asset is not registered"`